cmake/macros/macro_insource_setup_target.cmake

```diff
@@ -42,13 +42,11 @@ function(insource_setup_target _target _build)
       )
   endif()
 
-  target_include_directories(${_target}
-    PRIVATE
-      "${CMAKE_BINARY_DIR}/include"
-      "${CMAKE_SOURCE_DIR}/include"
-    SYSTEM PRIVATE
-      ${DEAL_II_BUNDLED_INCLUDE_DIRS}
-      ${DEAL_II_INCLUDE_DIRS}
+  target_include_directories(${_target} SYSTEM PRIVATE
+    "${CMAKE_BINARY_DIR}/include"
+    "${CMAKE_SOURCE_DIR}/include"
+    ${DEAL_II_BUNDLED_INCLUDE_DIRS}
+    ${DEAL_II_INCLUDE_DIRS}
```

Comment:
Why did you change this? Doesn't that imply that we don't see warnings from our headers when compiling examples?

Reply:
Good catch - this commit should have never made it into this PR in its current form. Guess it was late :sweat_smile: I have dropped it.